app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -117,6 +117,8 @@ def product_details(products)
   end
 
   def transport_type(question)
+    return [] unless session[question]
```

Comment:
Looks good. Since question will always be "transport_type", perhaps we could remove the arguments for this method and write `session["transport_type"]` here.

One could also do `Array(session["transport_type"]).flatten.to_sentence` below

Reply:
Good points.  Have updated.